pkg/test/framework/components/istio/operator.go

```diff
@@ -399,7 +400,7 @@ func installRemoteConfigCluster(i *operatorComponent, cfg Config, cluster resour
 	scopes.Framework.Infof("setting up %s as config cluster", cluster.Name())
 	// TODO move values out of external istiod test main into the ConfigClusterValues defaults
 	// TODO(cont) this method should just deploy the "base" resources needed to allow istio to read from k8s
-	installSettings, err := i.generateCommonInstallSettings(cfg, cluster, configIopFile)
+	installSettings, err := i.generateCommonInstallSettings(cfg, cluster, cfg.ConfigIOPFile, configIopFile)
```

Comment:
We should eventually add some methods to `resource.Cluster` like `IsConfigCluster` rather than leaving that on the env. If we had that, we wouldn't have to change the signature here, and we could just check that inside of `generateCommonInstallSettings`. Not needed in this PR, just something I've meant to add for a while.

Reply:
yep, Let me clear this PR out first. We can address this later.